dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/WindowCallbackWrapper.kt

```diff
@@ -102,6 +101,35 @@ internal class WindowCallbackWrapper(
 
     // endregion
 
+    // region Internal
+
+    private fun handleRemoteControlActionEvent() {
+        windowReference.get()?.currentFocus?.let {
+            val attributes = mutableMapOf<String, Any?>(
+                RumAttributes.ACTION_TARGET_CLASS_NAME to it.targetClassName(),
+                RumAttributes.ACTION_TARGET_RESOURCE_ID to resourceIdName(it.id)
+            )
+            val targetName = resolveTargetName(interactionPredicate, it)
+            GlobalRum.get().addUserAction(RumActionType.CLICK, targetName, attributes)
+        }
+    }
+
+    private fun handleBackEvent(event: KeyEvent) {
+        // TODO RUMM-495 add a BACK action to the json schema
```

Comment:
While you're here, the BACK event does exist, it might be a good opportunity to actually resolve this